File: demo/src/main/java/com/gerp83/slimadaptertest/ViewHolder1.java
package com.gerp83.slimadaptertest;

import android.view.View;
import android.widget.TextView;
import com.gerp83.slimadapter.SlimViewHolder;

/**
 * Created by gerp83
 */

public class ViewHolder1 extends SlimViewHolder{

    private TextView textView;

    public ViewHolder1(View itemView) {
        super(itemView);
        textView = itemView.findViewById(R.id.textView);
    }

    @Override
    public void fill(int position, Object data) {
        textView.setText("Text");
    }


}

File: demo/src/main/java/com/gerp83/slimadaptertest/ViewHolder1.kt
package com.gerp83.slimadaptertest

import android.view.View
import android.widget.TextView
import com.gerp83.slimadapter.SlimViewHolder

class ViewHolder1(itemView: View): SlimViewHolder(itemView) {

    private var textView: TextView = itemView.findViewById(R.id.textView)

    override fun fill(position: Int, data: Any) {
        textView.text = "Text"
    }

}